asp_interactive_configuration/DOC.md

```diff
@@ -130,17 +138,72 @@ The user input corresponds to the current partial configuration `C` that is bein
+
+### Usage
+
+When an attribute `ATTR` is of type `int`, their value is defined the `fclingo` variable `ooasp_attr_fvalue(ATTR,O)`, which will be associated to the value of `ATTR` for object `O`.
+These values can be compared using the theory symbol `&fsum`. For instance, in the following rule we use `&fsum{ooasp_attr_fvalue(A,new_object)}<MIN` to state that the value of `A` is smaller than `MIN` leading to a constraint violation. Notice that the variable `MIN` must be defined in a positive literal of the body.
+
+```
+ooasp_cv(value_outside_of_range,new_object,"Value for {} outside min range {}",(A,MIN)) :-
+	 ooasp_attr(C,A,int),
+	 ooasp_isa(C,new_object),
+	 &fsum{ooasp_attr_fvalue(A,new_object)}<MIN,
+	 ooasp_attr_minInclusive(C,A,MIN).
+```
+
+To check if a value is defined one can do so by checking if it is equal to itself. In the case of the next rule, if no value is defined, then the constraint violation is inferred.
+
+```
+ooasp_cv(no_value,new_object,"Missing value for {}",(A,)) :-
+	ooasp_attr(C,A,int),
+	ooasp_isa(C,new_object),
+	not &fsum{ooasp_attr_fvalue(A,new_object)}=ooasp_attr_fvalue(A,new_object).
+```
+
+These type of attributes must be treated differently to the rest, therefore they require rules like to ones above to handle each case. Bellow we show the difference of writing a constraint violation using `ooasp_attr_value/3` and `ooasp_attr_fvalue/2`. Notice that in the second case, we directly compare the values by summing them up.
+
+#### `ooasp_attr_value/3`
+```
+ooasp_cv(nrpassengers_neq_nrseats_plus_standingroom,new_object,"Number of passenges not adding up",(new_object,)):-
+    ooasp_isa(wagon,new_object),
+    ooasp_attr_value(nr_passengers,new_object,NRP),
+    ooasp_attr_value(nr_seats,new_object,NRS),
+    ooasp_attr_value(standing_room,new_object,S),
+    NRP!=NRS+S.
+```
+
+#### `ooasp_attr_fvalue/2`
+```
+ooasp_cv(nrpassengers_sum,new_object,"Number of passenges not adding up",(new_object,)):-
+    ooasp_isa(wagon,new_object),
+    &fsum{ooasp_attr_fvalue(nr_seats,new_object);
+          ooasp_attr_fvalue(standing_room,new_object)}!=
+        ooasp_attr_fvalue(nr_passengers,new_object).
+```
+
+### Interactivity
+
+The assigned values for these attributes are handled internally by the `fclingo `system and are only accessed after the models are found. Therefore, they are not considered in any brave or cautious consequences. This means that we do not know in advanced the possible values an attribute can take for a partial configuration. Interactively, this can be solve in different ways, such as using a slider instead of a dropdown or some other numerical input. For convenience, we keep the usage in the jupyter notebooks with dropdowns and add all possible values for the attribute, even though they might create an invalid configuration. Invialid configurations can still be checked and fixed in the UI. 
```

Comment:
"in advanced" -> "in advance"
"be solve" -> "be solved"
"Invialid" -> "Invalid"

Could you please use a spell checker in the future?